grpc/diatheke.proto

```diff
@@ -257,8 +257,21 @@ message ReplyAction {
 // This action indicates that the client application should
 // transcribe the user's input.
 message TranscribeAction {
-    // The ASR model to use for transcription
-    string cubic_model_id = 1;
+    // The ID of the transcribe action, which is useful to
+    // differentiate separate transcription tasks within a
+    // single sesssion.
+    string id = 1;
+
+    // The ASR model to use for transcription.
+    string cubic_model_id = 2;
+
+    // The Diatheke model where this transcribe action is
+    // defined. If empty, the server will not be able to
+    // automatically close the transcribe stream based on
+    // conditions defined in the Diatheke model. In that
```

Comment:
By "conditions defined in the Diatheke model", do we mean that diatheke will also be listening for keywords like "end transcription" or "done with comment" or what have you in order to stop transcribing?  If so, we might want to make that clearer in the documentation comment.

Reply:
That is exactly what I mean. I wasn't more specific in the comment because I wasn't 100% sure exactly what form those conditions would take. After having thought a little more about it, I think for a first run there will only be two - a timeout (no speech for n seconds) or an "end-transcription" intent, so I can make that clearer in the comment.